.github/copilot-instructions.md

```diff
@@ -191,6 +200,9 @@ FINAL ACTIONS:
    "Use the azure-rest-api-specs agent to handle the rest of the process and provide it the pull request."
 ```
 
+### STEP 8: RELEASE READINESS CHECK
+/check-package-readiness.prompt.md python language
```

Comment:
I understand this line is to hint agent follow check-package-readiness.prompt.md to check readiness for python package. Just curious that why we use format like `/xxx.md xx `? Is there any guidance about the prompt format?